ui/app/views/blocker/configure.html

```diff
@@ -35,6 +36,7 @@ <h1 class="page-header">Blockers Configuration</h1>
                                 <td>{{blocker.name}}</td>
                                 <td>{{allServices[blocker.serviceId].name || "*"}}</td>
                                 <td>{{allEnvironments[blocker.environmentId].name || "*"}}</td>
+                                <td>{{getStack(blocker.stackId)}}{{blockersStacks[blocker.stackId].name || "*"}}</td>
```

Comment:
You will be able to remove getStack call here after you'll index allStacks variable, and get you stack name  like this:
allStacks[blocker.stackId].name